packages/noble/src/sequence-search.ts

```diff
@@ -0,0 +1,78 @@
+import { FullViewingKey } from '@penumbra-zone/protobuf/penumbra/core/keys/v1/keys_pb';
+import { NobleClientInterface, NobleRegistrationResponse } from './client';
+
+// Search space (sequence number) is 2 bytes wide
+export const MAX_SEQUENCE_NUMBER = 65535;
+
+// Perform binary search to find the earliest unused noble sequence number
+export const getNextSequence = async ({
```

Comment:
Is there a reason the binary search for obtaining a noble sequence number wasn't implemented as a wasm helper?

Reply:
Because we don't really have any client fetching infra in wasm for making grpc or http requests. We could investigate this, but I'm not sure if it would add value here. Do you disagree?

Reply:
iirc, I thought core exposes an external rust interface that handles the networking logic that wasm can consume? maybe I'm wrong, but I don't disagree that it doesn't necessarily add extra value here.

Reply:
Given the use of tokio, there is not a wasm-compatible request helper I know of.